stdlib/public/stubs/Stubs.cpp

```diff
@@ -140,12 +140,21 @@ static int swift_snprintf_l(char *Str, size_t StrSize, locale_t Locale,
 #endif
 
 template <typename T>
+static int swift_floatingPointToStringPrecision(bool Debug) {
```

Comment:
As far as I understand the convention, methods are defined before being called(?)